scripts/exgdas_global_marine_analysis_prep.py

```diff
@@ -357,13 +357,16 @@ def find_bkgerr(input_date, domain):
     logging.info(f"using default SABER blocks yaml")
     os.environ['SABER_BLOCKS_YAML'] = os.path.join(gdas_home, 'parm', 'soca', 'berror', 'saber_blocks.yaml')
 
+# substitute templated variables in the var config
 logging.info(f"{config}")
-varconfig = YAMLFile(path=var_yaml_template)
+varconfig = ufsda.yamltools.DAYAMLFile(path=var_yaml_template)
```

Comment:
This is a step backwards IMO, we are going towards the `pygw` YAML tools and this goes back to the old `ufsda` tools...

Reply:
I don't see why @CoryMartin-NOAA , we're clearly in need of DA specific yaml manipulation tools, so a subclass of YAMLFile seems appropriate.